test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetIVsApiTests/Utils.cs

```diff
@@ -47,7 +48,39 @@ public static SimpleTestPackageContext CreatePackage(string packageName, string
             return package;
         }
 
-        public static bool IsPackageInstalledInAssetsFile(NuGetConsoleTestExtension nuGetConsole, string projectPath, string packageName, string packageVersion)
+        public static void AssertPackageIsInstalled(NuGetApexTestService testService, ProjectTemplate projectTemplate, ProjectTestExtension project, string packageName, string packageVersion)
+        {
+            switch (projectTemplate)
+            {
+                case ProjectTemplate.ClassLibrary:
```

Comment:
there needs to be a default. i'll let you decide if it's packages.config or PackageReference.

Reply:
I'd rather not rely on the project template as a switch, as we want Legacy Package Reference tests here as well.

ClassLibrary on it's own does not rely have an affiliation to PR or PC, it's a decision we can make based on the configuration for the package management type.

Reply:
@nkolev92 how do you suggest make this automation to look at assets file for PR and use IVS apis for PC without using the template as a base?

Reply:
The way it worked before is just fine. 

Just look for the assets file, if not default to PR.

Reply:
I'm open to other ideas which still enable the Legacy Package Reference scenario ofc :)

Being able to add scenario tests like that with Apex is important.